include/miopen/miopen.h

```diff
@@ -111,6 +111,12 @@ typedef enum
     miopenStatusVersionMismatch = 10, /*!< Version mismatch of the supplied binary data argment. */
 } miopenStatus_t;
 
+typedef enum
+{
+    miopenF8RoundingModeStandard   = 0,
+    miopenF8RoundingModeStochastic = 1,
```

Comment:
[Recommendation] Please add doc - explain what is standard mode and what is stochastic one.

Reply:
This represents how numbers are rounded from a higher precision type to FP8

Reply:
@JehandadKhan Thanks! I am guessing the same but what I would like to see this in doxygen doc. Does it sound reasonable?